hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -1312,6 +1319,46 @@ public double getInflightReplicationLimitFactor() {
       return inflightReplicationLimitFactor;
     }
 
+    @Config(key = "inflight.off-peak.hour",
+        type = ConfigType.STRING,
+        defaultValue = "-1",
```

Comment:
```suggestion
    @Config(key = "inflight.off-peak.hour",
        type = ConfigType.STRING,
        defaultValue = "",
```

nit: since this is a `String` config, I think it is more reasonable to use an empty string as the **off** switch.

`-1` is usually for `Integer` configs.